### PR TITLE
Signup Flow: Fix checking/unchecking of boxes for what 'check nearby lgas'

### PR DESCRIPTION
**Pull request by karlwestin**
WIP: Need to look at 'Edit App Config' as well

### CODE REVIEW

**Comment by karlwestin** [comment]
ok @tlvince i think this is ready to go now. its really just an attempt to restore the app functionality to a working state (namely the 'select nearby lgas' in the signup flow)

File: app/scripts/controllers/app-config.js [hunk 1]
```diff
@@ -324,11 +339,19 @@ angular.module('lmisChromeApp')
     };
 
     $scope.onLgaSelection = function(lga) {
+      var uuid = lga.uuid;
```
Comment (tlvince) [inline app/scripts/controllers/app-config.js]: Isn't this a repeat of line(s) 174 etc.?

Comment (karlwestin) [inline app/scripts/controllers/app-config.js]: ok let's refactor a little harder!

**Comment by tlvince** [comment]
LGTM, besides the comment, which can be its own task.

**Comment by karlwestin** [comment]
alright, let's have another look now

**Comment by tlvince** [comment]
Could you rebase this? Sorry

**Comment by karlwestin** [comment]
alright fixed :)

File: package.json [hunk 2]
```diff
@@ -2,8 +2,12 @@
   "name": "lmis-chrome",
   "version": "0.11.4",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "concat-map": "0.0.1"
```
Comment (tlvince) [inline package.json]: Seems unused?

File: package.json [hunk 3]
```diff
@@ -2,8 +2,12 @@
...
+  },
   "devDependencies": {
+    "balanced-match": "^0.2.0",
+    "cca": "0.4.3",
```
Comment (tlvince) [inline package.json]: Has been removed in #45

File: package.json [hunk 4]
```diff
@@ -2,8 +2,12 @@
   "name": "lmis-chrome",
   "version": "0.11.4",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "concat-map": "0.0.1"
+  },
   "devDependencies": {
+    "balanced-match": "^0.2.0",
```
Comment (tlvince) [inline package.json]: Unused?

File: karma.conf.js [hunk 5]
```diff
@@ -20,8 +20,9 @@ module.exports = function(config) {
     files: bowerJS.concat([
       'app/scripts/**/*.js',
       'test/mock/**/*.js',
-      'test/spec/{controllers,directives}/*.js',
-      'app/views/templates/*.html'
+//      'test/spec/{controllers,directives}/*.js',
```
Comment (tlvince) [inline karma.conf.js]: The controllers/directives specs pass for me. Any reason why you disabled them?

**Comment by tlvince** [comment]
This does a GET `facilities/by_lga` each time a nearby LGA is selected. Should it?

screen shot 2015-06-10 at 15 07 53

File: package.json (hunk 2 above)
Comment (karlwestin) [inline package.json]: i needed it to run the tests, couldn't get rid of it!
same w/ balanced-match!

**Comment by karlwestin** [comment]
about the GET request, i read that as the intention of the original code, so it seems to me it should.
I've fixed the other comments :)

**Comment by tlvince** [comment]
Yeah, looks like it. Seems like a potential performance improvement.

**Comment by tlvince** [comment]
LGTM. Nice work.

**Comment by tlvince** [comment]
Oh, could you commit the translations? ` M po/template.pot`

File: package.json (hunk 2 above)
Comment (tlvince) [inline package.json]: Oh? Couldn't see it `required` anywhere. Anyway, can it go in dev deps?

**Pull request merged**